_data/harnesses/kafka-connect-datagen-local/kafka.yml

```
@@ -0,0 +1,79 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: Dockerfile
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/make-dockerfile.adoc
+
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+            
+    - title: Create the connector
+      content:
+        - action: execute
+          file: tutorial-steps/dev/create-connector.sh
+          stdout: tutorial-steps/dev/outputs/create-connector.log
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/create-connector.adoc
+
+        - name: give Kafka Connect chance to create the connector
+          action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+        - action: execute
+          file: tutorial-steps/dev/check-connector.sh
+          stdout: tutorial-steps/dev/outputs/check-connector.log
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/check-connector.adoc
+
+        - name: give Kafka Connect further chance to get the data to the topic
+          action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+    - title: Consume events from the Kafka topic
+      content:
+      - action: execute_async
+        file: tutorial-steps/dev/harness-console-consumer-keys.sh
+        stdout: tutorial-steps/dev/outputs/consume-topic.log
+        render:
+          file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/consume-topic-key-value.adoc
+
+      - name: wait for consumer to read records
+        action: sleep
+        ms: 10000
+        render:
+          skip: true
+
+prod:
+  steps:
+    - title: Cleanup Docker containers
+      content:
+        - action: execute
+          file: tutorial-steps/dev/clean-up.sh
+          render:
+            skip: true
```

The clean-up step isn't displayed when running through the tutorial.  The `skip: true` part needs to be replaced with `file: <path>`
```suggestion
            tutorials/kafka-connect-datagen-local/kafka/markup/dev/clean-up.adoc 
```

---

@bbejeck ack, thank you.  I've implemented this in the latest commit (it needed `file:` addition.)